client/src/features/projectsV2/fields/AddProjectMemberModal.tsx

```diff
@@ -109,10 +103,36 @@ function AddProjectMemberAccessForm({
           onSubmit={handleSubmit(onSubmit)}
         >
           {result.error && <RtkErrorAlert error={result.error} />}
+          <div className="mb-3">
+            <Label className="form-label" for="addProjectMemberEmail">
+              User
+            </Label>
+            <Controller
+              control={control}
+              name="id"
+              render={({ field }) => {
+                const fields: Partial<typeof field> = { ...field };
+                delete fields?.ref;
+                return (
+                  <UserControl
+                    {...fields}
+                    className={cx(errors.id && "is-invalid")}
+                    data-cy="add-project-member"
+                    id="addProjectMember"
+                    onChange={(newValue: SingleValue<User>) =>
+                      field.onChange(newValue?.id)
+                    }
+                  />
+                );
```

Comment:
I do not see why it is necessary to modify the field object.
```suggestion
                return (
                  <UserControl
                    {...field}
                    className={cx(errors.id && "is-invalid")}
                    data-cy="add-project-member"
                    id="addProjectMember"
                    onChange={(newValue: SingleValue<User>) =>
                      field.onChange(newValue?.id)
                    }
                  />
                );
```